src/Files.App/UserControls/Widgets/FileTagsWidget.xaml.cs

```diff
@@ -51,6 +62,49 @@ private async void FileTagItem_ItemClick(object sender, ItemClickEventArgs e)
 				await itemViewModel.ClickCommand.ExecuteAsync(null);
 		}
 
+		private void Item_RightTapped(object sender, RightTappedRoutedEventArgs e)
```

Comment:
Idk but it feels to me as code duplication. Can that be avoided?

Reply:
I haven't found a way, the XAML elements are different